packages/frontend/src/pages/insight-editor/components/insight-editor-sidebar/sidebar-files.tsx

```diff
@@ -159,44 +175,70 @@ export const SidebarFiles = ({
     onTreeChanged(tree);
   };
 
+  const [isMobile] = useMediaQuery('(max-width: 768px)');
```

Comment:
Can you change this to `useBreakpoint()` instead?  Reason being, if we ever want to change the breakpoints, it would be great if everything used them instead of hardcoding widths in different places

```suggestion
  const isMobile = useBreakpoint({ base: true, md: false });
```

Reply:
Oh you're right, I didn't think of it